src/modules/ekf2/EKF/fake_pos_control.cpp

```diff
@@ -69,16 +71,20 @@ void Ekf::controlFakePosFusion()
 
 
 		const bool continuing_conditions_passing = !isHorizontalAidingActive();
+
 		const bool starting_conditions_passing = continuing_conditions_passing
+				&& ((Vector3f(getStateVariance<State::pos>()).xy().max() > pos_noaid_var)
+					|| (getTiltVariance() > sq(math::radians(3.f)))
```

Comment:
Or potentially could this be passed directly on tilt_align, with `tilt_align = (getTiltVariance() < sq(math::radians(3.f))`. Right now we never lose tilt_align.